example/MODULE.bazel

```diff
@@ -14,6 +14,7 @@ bazel_dep(name = "rules_proto", version = "6.0.0-rc2")
 bazel_dep(name = "rules_python", version = "0.26.0")
 bazel_dep(name = "buildifier_prebuilt", version = "6.3.3")
 bazel_dep(name = "platforms", version = "0.0.8")
+bazel_dep(name = "rules_kotlin", version = "1.9.0", repo_name = "io_bazel_rules_kotlin")
```

Comment:
is it possible to avoid renaming the repo?

Reply:
Yes, I can just use `rules_kotlin`.